content/U2F/.conf.json

```diff
@@ -1,10 +1,11 @@
 {
   "order": [
+    "../WebAuthn/Supporting_U2F_or_FIDO2_Security_Keys_on_iOS_or_iPadOS"
```

Comment:
this line needs to end with ,

Reply:
also: it will just be ignored, the order is for things present in the folder, not for including links to other parts.

Reply:
I will put in the whole document, then. And hope that if we need to update it, we remember that its twin is in the WebAuthn folder.